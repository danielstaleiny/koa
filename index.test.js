const fetchData = () => {
    return new Promise((resolve, reject) => resolve("success"));
};

test("the test", async () => {
    const data = await fetchData();
    expect(data).toBe("success");
});
