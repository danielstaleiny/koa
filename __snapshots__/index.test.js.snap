// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root route 1`] = `
Object {
  "id": "4",
  "text": "ni8ceee",
}
`;
